internal/server/v2/observation/golden/contained_in_2015_test.go

```diff
@@ -120,7 +120,7 @@ func TestFetchContainIn(t *testing.T) {
 	}
 	if err := test.TestDriver(
 		"FetchContainIn",
-		&test.TestOption{UseSQLite: true},
+		&test.TestOption{UseSQLite: true, CacheSVFormula: true},
```

Comment:
you don't need to enable this if this test does not test the formula part? And same for the other tests.

I expect to see golden file changes for this PR but seems no json have changed?

Reply:
I removed the option for now (will add later when I actually use it). Currently this just created the cache, but it's not referenced anywhere, so this change shouldn't affect any existing behavior